_posts/2024-09-15-labels-talk/labels-talk.Rmd

```diff
@@ -0,0 +1,158 @@
+We know that putting together a conference submission takes significant effort
+(especially putting together a video, during a pandemic) and we sincerely
+appreciate the time you've invested. We hope you find a venue for your proposal
+-- maybe even a future RStudio conference! Thank you again and we hope to still
+see you at rstudio::global(2021).
+
+**My take:** The "cringe" phrasing is problematic; posit::conf is really about 
+keeping a positive vibe. Also, maybe the conference committee didn't consider
+SAS/SPSS users to be a wide target audience. 
+
+
+## 2023 -- rejected
+
+**In R:** I was now working in clinical research, where my daily workflow
+includes importing SAS labelled data sets into R and doing subsequent wrangling 
+and reporting. I had also published two blog posts since my previous submission: [Leveraging labelled data in R](https://www.pipinghotdata.com/posts/2020-12-23-leveraging-labelled-data-in-r/) (related to the 2021 submission) and [The case for variable labels in R](https://www.pipinghotdata.com/posts/2022-09-13-the-case-for-variable-labels-in-r/)
+(related to this submission).
+
+**Title:** _The Case for Variable Labels in R_
+
+**Abstract:** Variable labels are valuable metadata providing brief descriptions of variables. Although historically associated with SAS, SPSS, or Stata data sets, variable labels can be embedded in R data frames as well. Integrating variable labels with your data requires minimal one-time effort, reduces the burden of referring to external data documentation, and alleviates the tension between succinct variable names for programming ease and longer descriptive text for interpretable output.  In this talk, I will showcase various R packages that allow you to import, assign, manage, and retrieve labels, as well as packages that automatically incorporate those labels in presentation ready outputs such as tables and figures.
+
+**Email response:**
+
+>Thank you for your submission to posit::conf(2023). 
+Unfortunately, your submission was not accepted for a talk this year. We received more excellent submissions than we can fit into the two days and we faced some tough decisions. In many cases, the deciding factor is creating a cohesive talk program, with a balance of topics.
+We know that putting together a conference submission takes significant effort (especially recording a video) and we sincerely appreciate the time and mental energy that you invested. Thank you again and we hope to still see you at posit::conf(2023).
+
+**My take:** Maybe it isn't exciting? Do I sound like a robot? I state what variable
+labels do but maybe not emphasize enough why you should be using them?
+
+## 2024 -- accepted
+
+**In R:** I had been doing more of the same since my previous submission. But
+perhaps at this time I even more firmly believed not just that creating labelled
+data is is workflow you _could_ incorporate but one that you _should_ incorporate.
+
+**Title:** _Context is King_
```

Comment:
in your comments on why it was accepted you might want to add that this title is way cooler than the other two titles!

Reply:
ha ok added a toned down version of this

I also tried to make the title more compelling...